modules/Twitter/src/main/java/totetmatt/gephi/twitter/utils/UserIdResolver.java

```diff
@@ -0,0 +1,27 @@
+package totetmatt.gephi.twitter.utils;
+
+import java.io.IOException;
+import java.util.Iterator;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class UserIdResolver {
+
+    static public long resolve(String userScreenName) throws IOException {
+        Document doc;
+        doc = Jsoup.connect(
+                "http://www.twitter.com/" + userScreenName.toLowerCase()).get();
+        Elements elems = doc.getElementsByAttribute("data-user-id");
+        Iterator<Element> els = elems.iterator();
+        while (els.hasNext()) {
+            Element el = els.next();
+            if (!el.attr("data-user-id").isEmpty()) {
+                return Long.parseLong(el.attr("data-user-id"));
+            }
+        }
+
+        return 0;
+    }
+}
```

Comment:
Long time that I didn't look actually at the API,and at that time it wasn't so permissive (drastic call limit mainly). One for the reason I've done this way was because you don't need to "connect" and it doesn't consume API limit. I'll have a look on the GET users/lookup and the interface with Twitter4J to check if it would be effectively better to use directly the API.